superbuild/README.md

```diff
@@ -33,11 +33,31 @@ $ cmake ${ISPC}/superbuild --preset os -DCCACHE=ON -DCCACHE_DIR=/tmp/shared-ispc
 $ cmake ${ISPC}/superbuild --preset os -DPREBUILT_STAGE2=/path/to/stage2-archive.tgz
```

Comment:
Here, in this `README` we use `${ISPC}`, while in other scripts we always use `${ISPC_HOME}`. I suggest using it consistently, i.e. use it here as well.